### PR TITLE
Add system logs to the app

### PR DESCRIPTION
**Pull request by licitdev**
## Scope

What's changed:

- Added system logs to the app
image

## Potential Risks / Drawbacks

- Needs UX feedback & improvement

## Review Notes / Questions

- The navigation button is hidden if websocket logs is not enabled

## TODO

- [x] Add translation strings for all text

---

App implementation for PR #23016 and related to issue #22482.

Closes SER-3

### CODE REVIEW

**Review by rijkvanzanten — COMMENTED** [review]
https://front-end.social/@bramus/112990110232760267 could be useful here @licitdev 🙂

**Comment by licitdev** [comment]
@rijkvanzanten Thank you for sharing, it's amazing! 😍 Logs are now fixed to the bottom even when not scrollable.

https://github.com/user-attachments/assets/42b1d0c7-5e59-46e0-bee0-b8f10b942c65

File: app/src/lang/translations/en-US.yaml [hunk 1]
```diff
@@ -1023,6 +1023,21 @@ no_data_in_flow: No data was produced or accessed within this Flow
...
 details: Details
+resume_streaming_logs: Resume Streaming Logs
+pause_streaming_logs: Pause Streaming Logs
+logs_session_resumed: Logs session resumed...
+logs_session_paused: Logs session paused...
+logs_stream_disconnected: Logs stream disconnected...
+log_detail_filter_paths: 'Filter Paths (eg: req.method, res.statusCode)'
+logs_beginning: This is the beginning of your logs session...
+logs_waiting: 'Awaiting more logs...'
+logs_unread_count: '{count} UNREAD'
```
[the remark this answers is not in the document]
Comment (licitdev) [reply app/src/lang/translations/en-US.yaml]: Updated in https://github.com/directus/directus/pull/23066/commits/e2907ccc36863c466a6be53481a792c2b4f7c3b6

File: app/src/modules/settings/routes/system-logs/components/logs-display.vue [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+			<div class="notice">{{ t('logs_beginning') }}</div>
+		</template>
+		<template #after>
+			<div v-if="streamConnected" class="notice">{{ t('logs_waiting') }}</div>
+		</template>
+		<template #default="{ item, index, active }">
+			<dynamic-scroller-item
+				:item="item"
+				:active="active"
+				:size-dependencies="[item.data.msg]"
```
[the remark this answers is not in the document]
Comment (licitdev) [reply app/src/modules/settings/routes/system-logs/components/logs-display.vue]: Great catch, this is a remnant of the initial implementation.
Updated in https://github.com/directus/directus/pull/23066/commits/67faa51c5346af6decf58860d8ec293baf5c313d

**Review by hanneskuettner — APPROVED** [review]
Amazing work, @licitdev! Love everything about it. LGTM 🎸

**Pull request merged**